examples/FluentUI.Demo.Shared/Pages/Lab/Highlighter/Highlighter.razor

```diff
@@ -0,0 +1,15 @@
+@namespace FluentUI.Demo.Shared
```

Comment:
Add @* Remember to replace the namespace below with your own project's namespace. *@ at top of file

Reply:
Done